### DISCUSSION TITLE
Should we reconsider our site directory structure?

### DISCUSSION THREAD

**Discussion by ayushn21** [bridgetownrb/bridgetown#271] [Ideas]
The number of config files/scripts we've got in our site root directory is getting a bit large! And it's just about to go up by one more after this PR gets merged:
https://github.com/bridgetownrb/bridgetown/pull/270

I wonder if there's any support in the community for `config` and `scripts` folders? Or am I just being a bit OCD about this haha!

This is what I'm proposing:

```
├── Gemfile
├── Gemfile.lock
├── package.json
├── yarn.lock
├── webpack.config.js
├── postcss.config.js
├── scripts 
|   ├── start.js
|   └── sync.js
├── config
|   ├── bridgetown.webpack.config.js
|   └── bridgetown.config.yml
├── plugins
├── src
├── frontend
```

What do you folks reckon?

**Comment by jaredcwhite** [comment]
I tend to think of a `scripts` folder as containing directly executable scripts in the Unix sense. I like the idea of a `config` folder though. Maybe those JS "scripts" could also go in `config`?

**Reply by ayushn21** [reply]
I did have the JS scripts in the `config` folder when I first wrote up the above post. Then I changed it because I didn't like it 😝. 

They're not really configurations though which is why I'm not that keen to be honest. Personally I tend to think of `bin` as the place of unix executables.

Since those files aren't really something a user would edit would could leave them in the root I suppose and just have a `config` folder. It'd be a handy place for plugins to put any config files as well. My content security policy plugin has a config file that just goes in the root at the moment. Can relocate that to `config`!
